sdk_lib/Dockerfile.sdk-import

```diff
@@ -43,11 +43,11 @@ RUN echo 'export PATH="$PATH:/usr/local/bin:/usr/local/sbin"' >> /home/sdk/.bash
 RUN echo "if [ -f /mnt/host/source/.env ]; then source /mnt/host/source/.env; fi" >> /home/sdk/.bashrc
 RUN echo "if [ -f /mnt/host/source/.sdkenv ]; then source /mnt/host/source/.sdkenv; fi" >> /home/sdk/.bashrc
 
-COPY --chown=sdk:sdk sdk_lib/sdk_entry.sh /home/sdk
-RUN chmod 755 /home/sdk/sdk_entry.sh
-
 # This should be a NOP; if you see packages being rebuilt
 #  it's likely that submodules and SDK tarball are out of sync
-RUN /home/sdk/sdk_entry.sh ./update_chroot --toolchain_boards="amd64-usr arm64-usr"
+RUN sudo su sdk -l -c "FEATURES='-ipc-sandbox -network-sandbox -pid-sandbox' /home/sdk/trunk/src/scripts/update_chroot --toolchain_boards='amd64-usr arm64-usr'"
+
+COPY --chown=sdk:sdk sdk_lib/sdk_entry.sh /home/sdk
+RUN chmod 755 /home/sdk/sdk_entry.sh
```

Comment:
This script has some commands that assume being root, I think we should either change `USER sdk:sdk` to `USER root:root` before this RUN step or change the script to use `sudo`

Reply:
Turns out user `root` is not in `/ect/passwd` or in `/etc/group` so `USER root:root` in `Dockerfile` fails. I'll update the `Dockerfile` to add `root`.

Reply:
(We still can't use `sdk_entry.sh` to call `update_chroot` because `update_chroot` needs to run before `setup_board` is called; however, `sdk_entry.sh` runs `setup_board` implicitly. The above is as good as it gets)